inst/rmd/h/pandoc/elevate-section-attrs.js

```diff
@@ -0,0 +1,14 @@
+$(function() {
+  $("div[class*='section level']").each(function(i, el) {
+    var $el = $(el);
+    var headers = $el.children().filter(":header");
+    for (var h = 0; h < headers.length; h++) {
+      var header = headers[h];
+      var attrs = header.attributes;
+      for (var a = 0; a < attrs.length; a++) {
+        $el.attr(attrs[a].name, attrs[a].value);
+      }
```

Comment:
Ok, yea that sounds right...Is it safe to say, with Pandoc < v2.8, that are sections headers *always* empty?

Reply:
Yes, I think so.

Reply:
> Is it safe to say, with Pandoc < v2.8, that are sections header attributes are _always_ empty?

Header attributes are empty for Pandoc < v2.8 when `--section-divs` is used. Otherwise, headers can have attributes. With Pandoc 2.7.3:

```bash
pandoc -t html4 <<< "# test"
```
gives
```html
<h1 id="test">test</h1>
```